bonsai/sdk/src/alpha.rs

```diff
@@ -482,11 +485,14 @@ impl Client {
             .send()?;
 
         if !res.status().is_success() {
+            if res.status().as_u16() == 404 {
```

Comment:
Instead of the as_u16 we could use: `if res.status() == reqwest::StatusCode::NOT_FOUND`
Not there is a `if res.status() == 204 {` in `get_image_upload_url` we should also change.

Reply:
[119475c](https://github.com/risc0/risc0/pull/1752/commits/119475cdb56ee15b18ca569a2a29c61d3b5467c2)